app/models/miq_server.rb

```diff
@@ -46,7 +46,7 @@ class MiqServer < ActiveRecord::Base
   STATUSES_ACTIVE  = [STATUS_STARTING, STATUS_STARTED]
   STATUSES_ALIVE   = STATUSES_ACTIVE + [STATUS_RESTARTING, STATUS_QUIESCE]
 
-  RESTART_SCRIPT = Rails.root.join("vmdb_restart").freeze
+  RESTART_EXIT_STATUS = 123
```

Comment:
Hmm, I wonder, should we be using one of the signal error exit codes (128+N)

apache uses SIGHUP for restart and SIGUSR1 for graceful restart
http://modperlbook.org/html/5-3-1-kill-Signals-for-Stopping-and-Restarting-Apache.html

If I `kill -1 26995` (SIGHUP) an empty script that sleeps:

```
[root@localhost ~]# ./test.sh
Hangup
[root@localhost ~]# echo $?
129
```

If I use SIGUSR1: `kill -10 27038`
```
[root@localhost ~]# ./test.sh
User defined signal 1
[root@localhost ~]# echo $?
138
```